### DISCUSSION TITLE
Open weather map values not always shown

### DISCUSSION THREAD

**Discussion by prutsky** [AllskyTeam/allsky-modules#83] [General]
I added the open weather data but I see the values are altering shown and not shown see this timelapse as an example:

http://www.harkbroek.nl/allsky/videos/allsky-20240208.mp4

Default the read-out is 600 seconds. I set it to 10 minutes, but didn't help (maybe I had too many requests).

Any thoughts why the values are off sometimes?

**Comment by Alex-developer** [comment]
That looks to me like its bouncing off of the request limit so I would try the following.

- Disable the OW module for 24 hours
- Ensure its only set to get data every 10 mins
- Re enable the module

Alex

**Comment by prutsky** [comment]
I noticed the "Expire time" was lower than the "Read every" time. Thought I used the default values, but these would probably by right.

Anyway, 600 seconds "Read every" and 1200 seconds " Expire time" results in nog more flickering :-)

**Reply by Alex-developer** [reply]
Glad you got it working

**Comment by Alex-developer** [comment]
> I noticed the "Expire time" was lower than the "Read every" time. Thought I used the default values, but these would probably by right.
> 
> Anyway, 600 seconds "Read every" and 1200 seconds " Expire time" results in nog more flickering :-)
